api/v1/storagecluster_types.go

```diff
@@ -244,26 +244,6 @@ type ExternalStorageClusterSpec struct {
 	//+kubebuilder:validation:Enum=ocs;rhcs
 	// StorageProviderKind Identify the type of storage provider cluster this consumer cluster is going to connect to.
 	StorageProviderKind ExternalStorageKind `json:"storageProviderKind,omitempty"`
```

Comment:
We dont need storageProviderKind also anymore?

Reply:
lets keep it with `rhcs` mode also it might help in the future to add new functionalities

Reply:
there is no reference to this, so I removed it. Also, in case required in the future, we can add it later.

Reply:
lets keep it for now as am not sure if someone is using this field other than managed service

Reply:
@Madhu-1 The default was always RHCS, so no RHCS deployment was using it. And the only other value is ODF which now becomes irrelevant. I think a quick search on the codebase will assure you that this is not even looked at. If I am mistaken here we can keep it

Reply:
Yes you are correct, it's not used anywhere to be on the safer side we are not removing the Field from the Spec to avoid the foreseen problem, we can also remove it later if it's not used by others after some confirmation.